### ISSUE TITLE
Stock Detail Page

### ISSUE THREAD

**Issue by MattGeimer** [CS196Illinois/Finance#40]
Related: #21 

Add a graph to the stock detail page, and make the page take in data to generate the rest of the page (string for name, double for price, array of doubles for historical prices to generate graphs, etc).

Also, feel free to pair program (have one person screen share what they're doing). It's actually fairly efficient. (Note that if you do this, just make a not of it in the pull request and only commit to one of your branches)

**Comment by MattGeimer** [comment]
Ran into issues embedding graphs into Tkinter window. @jflemi9

**Comment by MattGeimer** [comment]
@coolestinCS125 @PatinaShi614 For this week:
- Meet with James to learn how he embedded the graph into the home page
- Meet with @sajay94 and/or @arahim2 to learn about the helper functions and how to use them
- Meet with @YZY000403 to learn about the yFinance helper functions and how to use them

- Use the yfinance helper functions to gather data about a specific stock
- Use the JSON helper functions to find the qty of stocks the current user currently owns
- Embed a graph into your stock detail page
- Implement the buy/sell functions using the JSON helper functions and update the tkinter view accordingly

**Comment by jflemi9** [comment]
Hey @coolestinCS125  @PatinaShi614 send me a message over discord when you want to meet and we can set something up. If you want to take a look at things earlier I learned most of my stuff from a genius that I found on YouTube. I linked the playlist below. I especially recommend watching part 6 and part 7 which go over the graphing with matplotlib in tkinter. 
Anyways yeah its a good series again message me when you want to. Cheers


https://www.youtube.com/playlist?list=PLQVvvaa0QuDclKx-QpC9wntnURXVJqLyk
